packages/uniforms-antd/src/BoolField.js

```diff
@@ -6,8 +6,8 @@ import filterDOMProps from 'uniforms/filterDOMProps';
 
 import wrapField from './wrapField';
 
-const Bool = props =>
-    wrapField(props, (
+const Bool = ({labelCol, wrapperCol, ...props}) =>
```

Comment:
In #229 we've added `colon` and `required` - these also should be treated like this. By the way, we should find a better way to handle these... How about `filterDOMProps.register` them?